core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -933,18 +942,18 @@ $(document).ready(function () {
                 if (accumulatorTable.length == 0) {
                     $("#accumulator-update-table").hide();
                 } else {
-                    taskTableSelector.column(18).visible(true);
+                    taskTableSelector.column(19).visible(true);
                     $("#accumulator-update-table").show();
                 }
                 // Showing relevant stage data depending on stage type for task table and executor
                 // summary table
-                taskTableSelector.column(19).visible(dataToShow.showInputData);
-                taskTableSelector.column(20).visible(dataToShow.showOutputData);
-                taskTableSelector.column(21).visible(dataToShow.showShuffleWriteData);
+                taskTableSelector.column(20).visible(dataToShow.showInputData);
+                taskTableSelector.column(21).visible(dataToShow.showOutputData);
                 taskTableSelector.column(22).visible(dataToShow.showShuffleWriteData);
-                taskTableSelector.column(23).visible(dataToShow.showShuffleReadData);
-                taskTableSelector.column(24).visible(dataToShow.showBytesSpilledData);
+                taskTableSelector.column(23).visible(dataToShow.showShuffleWriteData);
```

Comment:
Why do we have this repeated twice?

Reply:
It was like this before. I have simply changed the column numbers. The flag showShuffleWriteData applies to both the columns "Write Time" as well as "Shuffle Write Size/Records".